src/routes/books.js

```diff
@@ -1,11 +1,11 @@
 import { Router } from 'express';
-import { Book } from '../models';
+import { Book, Review } from '../models';
 import catchAsync from '../lib/catchAsync';
 import { pick } from 'lodash';
 import Sequelize from 'sequelize';
 
 const router = Router();
-
+const Op = Sequelize.Op;
```

Comment:
You could also change the import line above to something like `import { Op }, Sequelize from 'sequelize`;`. I think...